### PR TITLE
Instruments Refactor and Electric Guitars

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Ports TG's instrument Refactor and Electric Guitars

- Adds in electric guitars
- Adds electric guitars to the cargo store
- Can craft electric guitars with crafting
- Can now craft violins with crafting
- Lowered the price of all instruments, at the cargo store, to 500
- Fixes violins not having an in-hand icon
- Service borgs can now shred with their electric guitar

The code is just reducing copy-pasta, more or less.

:cl: Fox McCloud
add: adds in electric guitars
add: can now craft violins and electric guitars
add: purchase electric guitars at the cargo store
add: service borgs now have an electric guitar, too
tweak: lowered the price of instruments, at cargo store, to 500
fix: Fixed violins not having an in-hand icon
/:cl:

### CODE REVIEW

File: code/modules/customitems/item_defines.dm [hunk 1]
```diff
@@ -246,7 +246,7 @@
 	new /obj/item/weapon/reagent_containers/food/drinks/cans/cola(src)
 
 
-/obj/item/device/guitar/jello_guitar //Antcolon3: Dan Jello
+/obj/item/device/instrument/guitar/jello_guitar //Antcolon3: Dan Jello
```
Comment (Fox-McCloud) [inline code/modules/customitems/item_defines.dm]: @tigercat2000 @Necaladun @TullyBurnalot 

This is a pathing change to a custom item; when this is merged the database will also have to be changed.

**Comment by Spacemanspark** [comment]
I've been wanting electric guitars for awhile now. Yesssss

File: code/game/objects/items/devices/instruments.dm [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+	attack_verb = list("played metal on", "serenaded", "crashed", "smashed")
+	hitsound = 'sound/effects/guitarsmash.ogg'
+	instrumentId = "guitar"
+
+/obj/item/device/instrument/eguitar
+	name = "electric guitar"
+	desc = "Makes all your shredding needs possible."
+	icon_state = "eguitar"
+	item_state = "eguitar"
+	force = 12
```
Comment (SamHPurp) [inline code/game/objects/items/devices/instruments.dm]: Surely an electric guitar's damage should go up to 11.

File: code/game/objects/structures/musician.dm [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
 	var/name = "Untitled"
 	var/list/lines = new()
-	var/tempo = 5					// delay between notes
+	var/tempo = 5			// delay between notes
 
-	var/playing = 0					// if we're playing
-	var/help = 0					// if help is open
-	var/repeat = 0					// number of times remaining to repeat
-	var/max_repeat = 10				// maximum times we can repeat
+	var/playing = 0			// if we're playing
```
Comment (SamHPurp) [inline code/game/objects/structures/musician.dm]: Again, boolean, `TRUE`/`FALSE`?

File: code/game/objects/items/devices/instruments.dm (hunk 2 above)
Comment (sothangel) [inline code/game/objects/items/devices/instruments.dm]: +1 for Spinal Tap reference.

**Comment by taukausanake** [comment]
I think I made guitar sprites once. Also make a dragon guitar. Maybe I can make it a fluff in the future

**Comment by alexkar598** [comment]
Yay I can destroy everybody ears with nyan cat on electric guitar.

**Pull request merged**